test/arrays/bugs/innerArrComp.good

```diff
@@ -0,0 +1 @@
+true true true
```

Comment:
I think when this test is working, it will need to have a linefeed on this line to pass.  A good habit to have when filing a future is to try creating a workaround version of the program that works as expected to verify that the .good file is correct (and then make sure to file the original program, not the workaround).  For example, in this case, the following programs are workarounds that could be used to generate/validate the .good file:

```chapel
var comp = [[true, true, true]];
writeln(comp);
```

or:

```chapel
var temp = [1,2,3] == [1,2,3];
var comp = [temp,];
writeln(comp);
```

Reply:
That definitely makes sense and is not a practice I have been following, thank you for pointing that out.